File: library/src/main/java/com/proxerme/library/enums/CommentSortCriteria.java
package com.proxerme.library.enums;

import com.squareup.moshi.Json;

/**
 * Enum holding the available sort options for comments.
 *
 * @author Ruben Gees
 */
public enum CommentSortCriteria {
    @Json(name = "rating")RATING,
    @Json(name = "")TIME
}

File: library/src/main/kotlin/me/proxer/library/enums/CommentSortCriteria.kt
package me.proxer.library.enums

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Enum holding the available sort options for comments.
 *
 * @author Ruben Gees
 */
@JsonClass(generateAdapter = false)
enum class CommentSortCriteria {

    @Json(name = "rating")
    RATING,

    @Json(name = "")
    TIME
}
